src/Data/Transformer/LoginTransformer.php

```diff
@@ -0,0 +1,19 @@
+<?php
+namespace App\Data\Transformer;
+
+use App\Data\Entity\Login;
+use App\Data\Transformer;
+
+class LoginTransformer extends Transformer
```

Comment:
Nope. I certainly can. This was just very simple. I'm implementing `JsonSerializable` on the classes, which means this is all run automatically when `json_encode` is called. Fractal is nice, but most of the code there is for processing the objects, which `JsonSerializable` takes care of for us.

Again, we can certainly change methods if we feel that Fractal works or is easier to handle at larger scale.